GraphNodeClassification.ipynb

```diff
@@ -0,0 +1,185 @@
+    "\n",
+    "https://githubtocolab.com/ThirdAILabs/Demos/blob/main/GraphNodeClassification.ipynb\n",
+    "\n",
+    "This notebook uses an activation key that will only work with this demo. If you want to try us out on your own dataset, you can obtain a free trial license at the following link: https://www.thirdai.com/try-bolt/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install thirdai --upgrade\n",
+    "!pip3 install scikit-learn\n",
+    "\n",
+    "# This activates the ThirdAI package with a key that is only good for this demo\n",
+    "import thirdai\n",
+    "\n",
+    "thirdai.licensing.activate(\"MWHE-FVNH-3XN3-KWXY-TFRA-X3VJ-U7FT-RLVR\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Download\n",
+    "We will use the demos module in the thirdai package to download the YelpCHI dataset. You can replace this step and the next step with a download method and a UDT initialization step that is specific to your dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from thirdai.demos import download_yelp_chi_dataset\n",
+    "\n",
+    "train_data_path, _, inference_samples, _ = download_yelp_chi_dataset()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# UDT Initialization\n",
+    "We can now create a UDT model by passing in the types of each column in the dataset and the target column we want to be able to predict. The input format for training UDT on a graph dataset is a typical CSV, where every row represents a single node in the graph. The bolt.types.node_id() column should be a unique integer identifying the node, while the bolt.types.neighbors() column should be a space separated list of node ids corresponding to the node's neighbors. For YelpChi, each node additionally has 32 numerical features (a \"feature vector\"), as well as a target column we wish to predict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from thirdai import bolt\n",
+    "\n",
+    "model = bolt.UniversalDeepTransformer(\n",
+    "    data_types={\n",
+    "        \"node_id\": bolt.types.node_id(),\n",
+    "        **{f\"col_{i}\": bolt.types.numerical([0, 1]) for i in range(32)},\n",
+    "        \"target\": bolt.types.categorical(),\n",
+    "        \"neighbors\": bolt.types.neighbors(),\n",
+    "    },\n",
+    "    target=\"target\",\n",
+    "    n_target_classes=2,\n",
+    "    integer_target=True,\n",
+    "    # Optional: you can remove the next line to increase AUC to ~0.93 at the\n",
+    "    # cost of increasing training time\n",
+    "    options={\"contextual_columns\": False},\n",
```

Comment:
contextual columns is true by default?

Reply:
I think you're right, this is by default false